user.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/json"
+  "encoding/binary"
+
+	"github.com/boltdb/bolt"
+)
+
+type membership struct {
+  DateStarted   string
+  DateEnded     string
+  MemberType    string
+}
+
+type payment struct {
+  Processing  bool
+  Processed   bool
+}
+
+type flags struct {
+	Date			string
+	FlaggerID	string
+}
+
+type User struct {
+  ID          		int
+  Username   			string
+  Password    		[]byte
+  Posts       		int
+  Email     		  string
+	Flags						[15]flags
+	Payment					payment
+	MembershipData	membership
```

Comment:
better to have all of this as one larger, flatter struct. It clutters and obfuscates things to break them apart when they always appear together.